website/docs/configurations.md

```diff
@@ -127,59 +127,59 @@ Options useful for writing tables via `write.format.option(...)`
-| [hoodie.meta.sync.client.tool.class](#hoodiemetasyncclienttoolclass)                                                                             | org.apache.hudi.hive.HiveSyncTool                            | Sync tool class name used to sync to metastore. Defaults to Hive.<br />`Config Param: META_SYNC_CLIENT_TOOL_CLASS_NAME`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| Config Name                                                                                                                                      | Default                                                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
why entire file is shown as changed?

Reply:
I think the formatter changed the length of one of the columns because the old config name was the longer than every other config

Reply:
configs have been updated so I reverted my changes from the configs pages